projects/Mallard/package.json

```diff
@@ -37,32 +37,34 @@
         "@react-native-community/async-storage": "^1.5.0",
         "@react-native-community/masked-view": "^0.1.1",
         "@react-native-community/netinfo": "^3.2.1",
+        "@react-native-community/push-notification-ios": "^1.0.2",
         "chalk": "^2.4.2",
         "convert-svg-to-png": "^0.5.0",
-        "deepmerge": "^3.2.0",
+        "deepmerge": "^4.0.0",
         "events": "^3.0.0",
         "invariant": "^2.2.4",
+        "jetifier": "^1.6.3",
         "kill-port": "^1.5.1",
         "moment": "^2.24.0",
         "node-fetch": "^2.6.0",
         "nodegit": "^0.24.3",
-        "query-string": "^6.8.1",
-        "react": "^16.8.6",
-        "react-native": "0.59.3",
+        "query-string": "^6.8.2",
+        "react": "^16.9.0",
+        "react-native": "0.60.4",
         "react-native-config": "^0.11.7",
         "react-native-device-info": "^2.3.2",
-        "react-native-gesture-handler": "^1.2.1",
+        "react-native-gesture-handler": "^1.3.0",
         "react-native-keychain": "^3.1.3",
-        "react-native-push-notification": "^3.1.3",
-        "react-native-screens": "^1.0.0-alpha.22",
+        "react-native-push-notification": "^3.1.8",
+        "react-native-screens": "^1.0.0-alpha.23",
         "react-native-sentry": "^0.43.2",
-        "react-native-status-bar-height": "^2.3.1",
-        "react-native-svg": "^9.4.0",
-        "react-native-webview": "^5.11.0",
-        "react-native-zip-archive": "^4.0.1",
-        "react-navigation": "3.11.0",
+        "react-native-status-bar-height": "^2.4.0",
+        "react-native-svg": "^9.6.2",
+        "react-native-webview": "^6.9.0",
```

Comment:
Can we find the breaking changes for this? Webview's are a bit shakey at the best of times!

Reply:
The changelog isn't particularly readable, but it doesn't seem to have any breaking changes.

Reply:
The changelog isn't particularly readable, but it doesn't seem to have any breaking changes.